### PR TITLE
Add bug report templates

### PR DESCRIPTION
**Pull request by valfirst**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+**To Reproduce**
+Steps to reproduce the behavior.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
```
Comment (uarlouski) [inline .github/ISSUE_TEMPLATE/bug_report.md]: does it make sense to add point about API bugs?

Comment (ikalinin1) [inline .github/ISSUE_TEMPLATE/bug_report.md]: I think we should also request environment info such as: 
- Vividus version
- OS/version where tests were running
- Java version
- Logs

Comment (valfirst) [reply .github/ISSUE_TEMPLATE/bug_report.md]: which sections do you propose to add?

Comment (uarlouski) [reply .github/ISSUE_TEMPLATE/bug_report.md]: never mind, all I wanted to say is spread across other points

**Pull request merged**